### PR TITLE
minor: fix commands at begining of development page

### PR DESCRIPTION
**Pull request by romani**

### CODE REVIEW

File: src/xdocs/beginning_development.xml [hunk 1]
```diff
@@ -70,23 +70,34 @@ git commit -m "commit message"
 git push origin my-new-check
            </source>
            5) Repeat steps 3-4 till development is complete<br/>
+           All additional commits, please <a href="https://davidwalsh.name/squash-commits-git">
+           squash to first</a>. PR should have only one commit.
+           <source class="wrap-content">
+git rebase -i master
```
[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs/beginning_development.xml]: it is rebase over local master, should be just "master"

File: src/xdocs/beginning_development.xml [hunk 2]
```diff
@@ -70,23 +70,34 @@ git commit -m "commit message"
...
            6) Update current branch and local master by pulling changes that were done<br/>
            by other contributors:
            <source class="wrap-content">
-git pull --rebase main master
+git pull main master
+git push origin master
+           </source>
+           7) Rebase your branch over your updated master
+           <source class="wrap-content">
+git rebase master
```
[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs/beginning_development.xml]: it is rebase over local master, should be just "master"

File: src/xdocs/beginning_development.xml [hunk 3]
```diff
@@ -70,23 +70,34 @@ git commit -m "commit message"
 git push origin my-new-check
            </source>
            5) Repeat steps 3-4 till development is complete<br/>
+           All additional commits, please <a href="https://davidwalsh.name/squash-commits-git">
+           squash to first</a>. PR should have only one commit.
```
[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs/beginning_development.xml]: done

File: src/xdocs/beginning_development.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rnveach) [reply src/xdocs/beginning_development.xml]: Why is it using local master? no one will update local master and they may not even keep it.

[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs/beginning_development.xml]: as agreed in chat, we will reconsider such steps in separate PR

**Pull request merged**